.tekton/argocd-agentctl-cli-pull-request.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  annotations:
+    build.appstudio.openshift.io/repo: https://github.com/rh-gitops-midstream/release?rev={{revision}}
+    build.appstudio.redhat.com/commit_sha: '{{revision}}'
+    build.appstudio.redhat.com/pull_request_number: '{{pull_request_number}}'
+    build.appstudio.redhat.com/target_branch: '{{target_branch}}'
+    pipelinesascode.tekton.dev/cancel-in-progress: "true"
+    pipelinesascode.tekton.dev/max-keep-runs: "3"
+    pipelinesascode.tekton.dev/on-cel-expression: >-
+      event == "pull_request" &&
+      target_branch == "main" &&
+      (
+        "sources/argocd-agent".pathChanged() ||
+        "clis/argocd-agentctl/***".pathChanged() ||
+        ".tekton/argocd-agentctl-cli-pull-request.yaml".pathChanged() ||
+        ".tekton/build-multi-platform-image.yaml".pathChanged() ||
+        ".tekton/tasks/***".pathChanged()
```

Comment:
```suggestion
```
You can remove this. We are currently not using any metadata generated by the custom task